thesis-gui/src/components/question-cards/definition-question-card/DefinitionQuestionCard.tsx

```diff
@@ -1,28 +1,42 @@
 import React from "react";
 
 import {
+    IonButton, IonButtons,
     IonCard, IonCardContent,
     IonCardHeader,
-    IonCardSubtitle, IonCardTitle
+    IonCardSubtitle, IonCardTitle, IonIcon
 } from '@ionic/react';
 import './DefinitionQuestionCard.css';
-import { CategoryType } from "../../../types/category-type";
+import {CategoryType} from "../../../types/category-type";
+import {flagOutline} from "ionicons/icons";
 
-export const DefinitionQuestionCard: React.FC<{ question: string, questionNumber: number, category: CategoryType }> = props => {
-    return (
-        <IonCard>
-            <IonCardHeader>
-                <IonCardSubtitle>
-                    { `Category: ${!!props.category ? props.category.name : ''}` }
-                </IonCardSubtitle>
-                <IonCardTitle>
-                    { `Question ${props.questionNumber || ''}` }
-                </IonCardTitle>
-            </IonCardHeader>
+export const DefinitionQuestionCard: React.FC<{
+    question: string,
+    questionNumber: number,
+    category: CategoryType,
+    flagDisabled: boolean,
+    flagQuestion: () => void
+}> = props => (
+    <IonCard>
+        <IonCardHeader>
+            <IonCardSubtitle>
+                {`Category: ${!!props.category ? props.category.name : ''}`}
+            </IonCardSubtitle>
+            <IonCardTitle style={{display: 'flex', justifyContent: 'space-between'}}>
+                {`Question ${props.questionNumber || ''}`}
+                <IonButtons>
+                    <IonButton
+                        disabled={props.flagDisabled}
+                        onClick={props.flagQuestion}
+                    >
+                        <IonIcon slot="icon-only" icon={flagOutline}/>
```

Comment:
I know I'm being picky 🫣but I think the **not outlined** icon is more visible

Reply:
Done